### ISSUE TITLE
ModelListType preview widget not showing name

### ISSUE THREAD

**Issue by jorenvh1** [sonata-project/SonataAdminBundle#5209]
Hi there

Whenever I select something (or add a new one) to a field of ModelListType I can see the previewed name inside the field. 

When I update the entity and the item is linked to the property, the previewed name disappears. Can anyone tell me if this is a bug or did I do something wrong?

This happens on Media items (files names don't show up) and also one to one relationships to other entities,...

The weird thing is that this works perfectly on my local development server but not on a staging env...

<img width="492" alt="screen shot 2018-08-24 at 09 44 26" src="https://user-images.githubusercontent.com/14889143/44578777-c2343580-a794-11e8-9a98-7bf8e81e868a.png">

**Comment by jorenvh1** [comment]
The problem for me was caused by the fact that these fields do an internal request. 
For some reason this request caused a `ConflictingHeadersException`. This error was silently ignored and this just caused the field to be empty. 
What I did to fix this is add the following line to my app.php:

`Request::setTrustedProxies(['127.0.0.1'], Request::HEADER_X_FORWARDED_ALL);`

Feels really weird to me that I have to explicitly allow localhost to be a trusted proxy though...
